events/wmt21.md

```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+#### Full results of the shared tasks: [***Findings of the 2021 Conference on Machine Translation (WMT21)***](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English (segment rating with document context)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| Czech→English | 77.8 | 0.111 | `Facebook-AI` |
+| German→English | 71.9 | 0.126 | `Borderline` |
+| Hausa→English | 74.4 | 0.248 | `Facebook-AI` |
+| Icelandic→English | 74.5 | 0.293 | `Facebook-AI` |
+| Japanese→English | 73.8 | 0.141 | `HW-TSC` |
+| Russian→English | 77.5 | 0.137 | `NVIDIA-NeMo` |
+| Chinese→English | 75.0 | 0.042 | `NiuTrans`|
+
+#### English → (segment ranking in a full document)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| English→Czech | 90.2 | 0.397 | `HUMAN-A` |
+| English→German | 83.3 | 0.266 | `Online-B` |
+| English→Hausa | 84.1 | 0.362 | `HUMAN-A` |
+| English→Icelandic | 88.1 | 0.872 | `HUMAN-A` |
+| English→Japanese | 86.4 | 0.430 | `Facebook-AI` |
+| English→Russian | 86.0 | 0.317 | `HUMAN-B` |
+| English→Chinese | 82.5 | 0.325 | `HUMAN-B`|
+| French→German | 87.7 | 0.088 | `Online-W`|
+| German→French | 87.9 | 0.160 | `Online-B`|
+
+#### French to/from German (segment rating in a full document)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| French→German | 87.7 | 0.088 | `Online-W`|
+| German→French | 87.9 | 0.160 | `Online-B`|
+
```

Comment:
several changes to kept consistency with the structure of yearly results in wmt articles